### PR TITLE
Add missing authorization directives to queries

### PR DESCRIPTION
**Pull request by tgorgol**
**Description**

Changes proposed in this pull request:

- Add missing directives to schema file
- Add authorization test suite to acceptance tests
- Disable secret subscription until https://github.com/kyma-project/kyma/issues/3412 gets resolved

**Related issue(s)**

#3062

### CODE REVIEW

File: tests/console-backend-service/internal/domain/k8s/secret_test.go [hunk 1]
```diff
@@ -211,20 +224,20 @@ func fixSecretsQuery() *graphql.Request {
...
-						name
-					}
-				}
-			}`
-	req := graphql.NewRequest(query)
-	req.SetVar("namespace", secretNamespace)
-
-	return req
-}
+//func fixSecretSubscription() *graphql.Request {
```
Comment (pkosiec) [inline tests/console-backend-service/internal/domain/k8s/secret_test.go]: Why is it commented?

File: tests/console-backend-service/internal/domain/k8s/secret_test.go [hunk 2]
```diff
@@ -267,31 +280,31 @@ func fixDeleteSecretMutation() *graphql.Request {
...
-		event, err := readSecretEvent(sub)
-		if err != nil {
-			return err
-		}
-		if expected.Type == event.Type && expected.Secret.Name == event.Secret.Name {
-			return nil
-		}
-	}
-}
+//func secretEvent(eventType string, secret secret) SecretEvent {
```
Comment (pkosiec) [inline tests/console-backend-service/internal/domain/k8s/secret_test.go]: Why is it commented?

File: tests/console-backend-service/internal/domain/shared/auth/suite.go [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+			case List:
+				a.testList(t, req)
+			case Create:
+				a.testCreate(t, req)
+			case Update:
+				a.testUpdate(t, req)
+			case Delete:
+				a.testDelete(t, req)
+			default:
+				log.Println("unknown operation type")
```
Comment (pkosiec) [inline tests/console-backend-service/internal/domain/shared/auth/suite.go]: `t.Fail()` would be better

File: tests/console-backend-service/internal/domain/k8s/secret_test.go (hunk 2 above)
Comment (tgorgol) [reply tests/console-backend-service/internal/domain/k8s/secret_test.go]: We are disabling secret subscriptions temporarily due to security reasons.

File: tests/console-backend-service/internal/domain/k8s/secret_test.go (hunk 1 above)
Comment (tgorgol) [reply tests/console-backend-service/internal/domain/k8s/secret_test.go]: We are disabling secret subscriptions temporarily due to security reasons.

**Pull request merged**